### PR TITLE
fix: unable to copy in safari whenever sending trigger in ClipboardButton

### PR DESCRIPTION
**Pull request by Elessar1802**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist

- [ ] The title of the PR states what changed and the related issues number (used for the release note).
- [ ] Does this PR require documentation updates?
- [ ] I've updated documentation as required by this PR.
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by AbhishekA1509 — DISMISSED** [review]

File: src/Common/ClipboardButton/ClipboardButton.tsx [hunk 1]
```diff
@@ -14,9 +14,9 @@
  * limitations under the License.
  */
 
-import { useState, useEffect, useCallback } from 'react'
-import Tippy from '@tippyjs/react'
+import { useState, useEffect, useRef } from 'react'
```
Comment (AbhishekA1509) [inline src/Common/ClipboardButton/ClipboardButton.tsx:17]: Suggestion: We can remove this file from eslint ignore

File: src/Common/ClipboardButton/ClipboardButton.tsx [hunk 2]
```diff
@@ -25,73 +25,87 @@ import { ReactComponent as Check } from '../../Assets/Icon/ic-check.svg'
...
+
     return (
-        <Tippy
-            className="default-tt"
-            content={copied ? copiedTippyText : 'Copy'}
-            placement="bottom"
-            visible={copied || enableTippy}
-            arrow={false}
+        <Tooltip
+            content={'Copy'}
```
Comment (AbhishekA1509) [inline src/Common/ClipboardButton/ClipboardButton.tsx:95]: Should be content="Copy" and why are we nesting tooltips? can we move this tooltip inside button or vice-versa?

**Review by AbhishekA1509 — CHANGES_REQUESTED** [review]

File: src/Common/ClipboardButton/ClipboardButton.tsx [hunk 3]
```diff
@@ -25,73 +25,87 @@ import { ReactComponent as Check } from '../../Assets/Icon/ic-check.svg'
...
+            alwaysShowTippyOnHover={!copied}
         >
             <button
```
Comment (AbhishekA1509) [inline src/Common/ClipboardButton/ClipboardButton.tsx:98]: Semantically we should not have a button inside a button

Comment (Elessar1802) [reply src/Common/ClipboardButton/ClipboardButton.tsx:98]: Can fix later. have added a TODO

**Review by AbhishekA1509 — DISMISSED** [review]

**Review by vivek-devtron — DISMISSED** [review]

**Review by vivek-devtron — APPROVED** [review]

**Review by vivek-devtron — APPROVED** [review]

**Pull request merged**